docs/assemblies/assembly-multi-tenancy-filter.adoc

```diff
@@ -0,0 +1,27 @@
+// file included in the following:
+//
+// assembly-built-in-filters.adoc
+
+[id='assembly-multi-tenancy-filter-{context}']
+= (Preview) Multi-tenancy filter
+
+[role="_abstract"]
+Kroxylicious’s Multi-tenancy filter presents a single Kafka cluster to tenants as if it were multiple clusters.
+Tenant-specific operations are isolated by prefixing resources with an identifier.
```

Comment:
I find the phrase `Tenant-specific operations` a bit miss leading as it implies there are operations on the Kafka cluster which are not `tenant-specific` which isn't true so personally I would make it 
```suggestion
Operations are isolated to a single tenant by prefixing resources with an identifier.
```